README.md

```diff
@@ -36,6 +36,8 @@ Read the [Roadmap](./ROADMAP.md).
 ## Quick start
 
 - __Pre-requirement__  
+
+golang version 1.5 or later
```

Comment:
Go environment. Currently a 64-bit version of go >= 1.5 is required.

seems better.